packages/react-liveness/src/components/FaceLivenessDetector/service/machine/index.ts

```diff
@@ -1189,8 +1189,8 @@ export const livenessMachine = createMachine<LivenessContext, LivenessEvent>(
 );
 
 const responseStreamActor = async (callback: StreamActorCallback) => {
-  const stream = await responseStream;
   try {
+    const stream = await responseStream;
```

Comment:
Fixes it, previously we weren't catching the error at all so it would not trigger our xstate error flow. Now since it is part of the try catch block it should fire an error if one occurs while connection is being established